tests/systemtests/system.go

```diff
@@ -331,6 +331,37 @@ func (s *SystemUnderTest) StopChain() {
 	s.ChainStarted = false
 }
 
+// StopSingleNode stops a validator node without stop the chain running
+func (s *SystemUnderTest) StopSingleNode() error {
+	if !s.ChainStarted {
+		return nil
+	}
+
+	s.pidsLock.RLock()
+	pids := maps.Keys(s.pids)
+	s.pidsLock.RUnlock()
+
+	p, err := os.FindProcess(pids[0])
+	if err != nil {
+		return err
+	}
+
+	// Kill the 1st node
+	return p.Kill()
```

Comment:
Do we really want to kill it that way? This sends iirc a SIGKILL, we may want to stop the node more gracefully with SIGTERM